controllers/mod/search.rb

```
@@ -0,0 +1,71 @@
+        page, page_size = page_params
+
+        resp = search(Ontology, query, options)
+
+        result = {}
+        acronyms_ids = {}
+        resp.each do |doc|
+          id = doc["submissionId_i"]
+          acronym = doc["ontology_acronym_text"] || doc["ontology_t"]&.split('/')&.last
+          next if acronym.blank?
+
+          old_id = acronyms_ids[acronym].to_i rescue 0
```

RuboCop (1.75.5)

[convention] Avoid using `rescue` in its modifier form.

(Style/RescueModifier)